mixin/thanos/alerts/sidecar.libsonnet

```diff
@@ -9,6 +9,19 @@
       {
         name: 'thanos-sidecar.rules',
         rules: [
+          {
+            alert: 'ThanosSidecarPrometheusDown',
+            annotations: {
+              message: 'Thanos Sidecar {{$labels.job}} {{$labels.pod}} cannot connect to Prometheus.',
```

Comment:
You're referring to a `job` and `pod` label in alert name, these have to be exposed by your alert expression down below at line 18.

Something like
`sum by (job, pod) (thanos_sidecar_prometheus_up{%(selector)s} == 0)`

Reply:
By default, when you search a metric, it would return all label. Is it necessary?

Reply:
Nevermind. you're right. It's not necessary in this your case. You already have a pod label.